vispy/app/backends/_qt.py

```diff
@@ -179,15 +185,21 @@ def _set_config(c):
     glformat.setGreenBufferSize(c['green_size'])
     glformat.setBlueBufferSize(c['blue_size'])
     glformat.setAlphaBufferSize(c['alpha_size'])
-    glformat.setAccum(False)
-    glformat.setRgba(True)
-    glformat.setDoubleBuffer(True if c['double_buffer'] else False)
-    glformat.setDepth(True if c['depth_size'] else False)
+    if hasattr(glformat, 'setAccum'):
```

Comment:
These toggles don't seem to exist. I think they are just automatically enabled when you set the size > 0.